mir_eval/transcription.py

```diff
@@ -264,21 +431,22 @@ def match_notes(ref_intervals, ref_pitches, est_intervals, est_pitches,
     return matching
 
 
-def precision_recall_f1(ref_intervals, ref_pitches, est_intervals, est_pitches,
-                        onset_tolerance=0.05, pitch_tolerance=50.0,
-                        offset_ratio=0.2, offset_min_tolerance=0.05,
-                        strict=False):
+def precision_recall_f1_overlap(ref_intervals, ref_pitches, est_intervals,
+                                est_pitches, onset_tolerance=0.05,
+                                pitch_tolerance=50.0, offset_ratio=0.2,
+                                offset_min_tolerance=0.05, strict=False):
     """Compute the Precision, Recall and F-measure of correct vs incorrectly
-    transcribed notes. "Correctness" is determined based on note onset, pitch
-    and (optionally) offset: an estimated note is assumed correct if its onset
-    is within +-50ms of a ref note and its pitch (F0) is within +- quarter tone
-    (50 cents) of the corresponding reference note. If with_offset is False,
-    note offsets are ignored in the comparison. It with_offset is True,
-    on top of the above requirements, a correct returned note is required to
-    have an offset value within 20% (by default, adjustable via the
-    offset_ratio parameter) of the ref note's duration around the ref note's
-    offset, or within offset_min_tolerance (50 ms by default), whichever is
-    larger.
+    transcribed notes, and the Average Overlap Ratio for correctly transcribed
+    notes (see :func:`average_overlap_ratio`). "Correctness" is determined
+    based on note onset, pitch and (optionally) offset: an estimated note is
+    assumed correct if its onset is within +-50ms of a ref note and its pitch
+    (F0) is within +- quarter tone (50 cents) of the corresponding reference
+    note. If with_offset is False, note offsets are ignored in the comparison.
```

Comment:
thanks for catching this, must have slipped through in the previous PR. will fix.